container/common/fsHandler.go

```diff
@@ -90,7 +90,7 @@ func (fh *realFsHandler) update() error {
 	fh.Lock()
 	defer fh.Unlock()
 	fh.lastUpdate = time.Now()
-	if fh.rootfs != "" && rootErr == nil {
+	if fh.rootfs != "" && rootErr == nil && extraErr == nil {
```

Comment:
I believe it is valid for fh.extraDir to be unset, which would cause GetDirUsage above to return an uninitialized UsageInfo struct.  Thankfully, the zero-value of the Bytes field is 0, so the current setup is technically correct.

I agree that it would be more readable to check the error, but we don't want to not set TotalUsageBytes if we get one.

Can we instead do a separate check on extraErr == nil, and then do `fh.usage.TotalUsageBytes += extraUsage.Bytes`?

Reply:
@dashpole 
Please take another look.

Reply:
Just remove the check for extraErr == nil here, and we are all good to go.

Reply:
Addressed, @dashpole